common/lib/xmodule/xmodule/course_module.py

```diff
@@ -682,7 +682,7 @@ class CourseFields(object):
         ),
         scope=Scope.settings,
         # Ensure that courses imported from XML keep their image
-        default="images_course_image.jpg"
+        default=""
```

Comment:
The thing is, this will break XML integration so I don't think this is the right way to go. I'm also not sure who's best to ask here.